### ISSUE TITLE
Code formatting issues with ofxCvContourFinder documentation pages

### ISSUE THREAD

**Issue by CorpusCallosum** [openframeworks/ofSite#821]
Hi, just noticed that the code formatting on these two pages is broken: 
- https://openframeworks.cc/documentation/ofxOpenCv/ofxCvContourFinder/#show_blobs
- https://openframeworks.cc/documentation/ofxOpenCv/ofxCvContourFinder/#show_nBlobs

But, looking at the source files, I didn't see any issues, so not sure what's causing the code formatting to break here:
https://raw.githubusercontent.com/openframeworks/ofSite/master/documentation/addons/ofxOpenCv/ofxCvContourFinder.markdown

**Comment by dimitre** [comment]
it seems to be using four tildes instead of three
here in github the default is three backticks \` with the language like 
\`\`\`c++

**Comment by CorpusCallosum** [comment]
Yes, I believe that is correct, it should be using 4 tildes, as that is described in the readme as the correct approach, and also if you check the other documentation pages source codes, they are also using 4x tildes. 

see: https://github.com/openframeworks/ofSite/blob/master/README.md

and for reference this page is using 4 tildes ~~~~ as an example: https://raw.githubusercontent.com/openframeworks/ofSite/master/documentation/math/ofMath_functions.markdown
